src/api/package-api.ts

```diff
@@ -15,8 +15,8 @@ class PackageApi {
         });
     }
 
-    public async exportPackage(rootPackageKey: string): Promise<Buffer> {
-        return await httpClientV2.downloadFile(`/package-manager/api/packages/${rootPackageKey}/export?newKey=${rootPackageKey}`);
+    public async exportPackage(rootPackageKey: string, version?: string): Promise<Buffer> {
+        return await httpClientV2.downloadFile(`/package-manager/api/packages/${rootPackageKey}/export?newKey=${rootPackageKey}${version ? `&version=${version}` : ""}`);
```

Comment:
can you look to use `querystring.stringify`to build parameters like we did in some other methods